exec/java-exec/src/main/java/org/apache/drill/exec/planner/physical/SingleMergeExchangePrel.java

```diff
@@ -93,6 +94,21 @@ public PhysicalOperator getPhysicalOperator(PhysicalPlanCreator creator) throws
     return creator.addMetadata(this, g);
   }
 
+  /**
+   * This method creates a new OrderedMux exchange if mux operators are enabled.
+   * @param child input to the new muxPrel or new SingleMergeExchange node.
+   * @param options options manager to check if mux is enabled.
+   */
+  @Override
+  public Prel constructMuxPrel(Prel child, OptionManager options) throws RuntimeException {
+    Prel outPrel = child;
+    if (options.getOption(PlannerSettings.ORDERED_MUX_EXCHANGE.getOptionName()).bool_val) {
```

Comment:
@HanumathRao I think the ordered_mux should be created when both mux and ordered_mux flags are enabled.  Users who disable the global 'mux' flag would very likely expect that all mux exchanges are disabled.

Reply:
@amansinha100  Thanks for the review . I have done the needed code changes to fix it. Please let me know if anything is required.